core/src/test/scala/tech/beshu/ror/unit/acl/factory/decoders/rules/auth/ExternalAuthorizationRuleSettingsTests.scala

```diff
@@ -678,8 +742,40 @@ class ExternalAuthorizationRuleSettingsTests
               |
               |  - name: GroupsService1
               |    groups_endpoint: "http://localhost:8080/groups"
+              |    auth_token_name: "user"
               |    auth_token_passed_as: QUERY_PARAM
               |    response_groups_json_path: "$..groups[?(@.name)].name"
+              |    response_group_ids_json_path: "$..groups[?(@.id)].id"
+              |""".stripMargin,
+          httpClientsFactory = mockedHttpClientsFactory,
+          assertion = errors => {
+            errors should have size 1
+            errors.head should be(DefinitionsLevelCreationError(Message(
+              "External authorization service 'GroupsService1' configuration cannot have the 'response_groups_json_path' and 'response_group_ids_json_path' attributes defined at the same time"
+            )))
+          }
+        )
+      }
+      "authorization service auth token name is not defined" in {
```

Comment:
I'd propose one more test where:
* `response_group_ids_json_path` is not defined
* `response_group_names_json_path` is defined

Reply:
1. Added
2. We cannot explicitly test `response_group_names_json_path`. because this property is hidden in the `ExternalAuthorizationService` internals. I added this field to tests using all available fields

Reply:
2. what about such primitive, but anyway effective method?

```scala
val ldapService = definitions.items.head
ldapService shouldBe a[CacheableLdapServiceDecorator]
val ldapServiceUnderlying = getUnderlyingFieldFromCacheableLdapServiceDecorator(ldapService.asInstanceOf[CacheableLdapServiceDecorator])
ldapServiceUnderlying shouldBe a[CircuitBreakerLdapServiceDecorator]
ldapServiceUnderlying.asInstanceOf[CircuitBreakerLdapServiceDecorator].circuitBreakerConfig shouldBe CircuitBreakerConfig(Refined.unsafeApply(10), Refined.unsafeApply(10 seconds))
ldapService.id should be(LdapService.Name("ldap1"))
```

Reply:
I added more assertions to tests